subsys/bluetooth/host/hci_core.h

```diff
@@ -243,6 +243,7 @@ struct bt_dev_le {
 #if defined(CONFIG_BT_ISO)
 	uint16_t		iso_mtu;
 	struct k_sem		iso_pkts;
+	uint32_t		iso_limit;
```

Comment:
The value stored here is the `rp->iso_max_num` which is a `uint8_t`. 

```suggestion
	uint8_t		iso_limit;
```